### ISSUE TITLE
syn 1.0.58 breaks build due to private api usage

### ISSUE THREAD

**Issue by AsumFace** [zombodb/pgx#71]
1.0.58 includes this commit https://github.com/dtolnay/syn/commit/957840eba2c7f11c95e0227760d78dc481a91de7 that breaks builds with the following error:
```
error[E0432]: unresolved import `syn::export`
  --> <user>/.cargo/registry/src/github.com-1ecc6299db9ec823/pgx-utils-0.1.11/src/lib.rs:12:10
   |
12 | use syn::export::TokenStream2;
   |          ^^^^^^ could not find `export` in `syn`

error[E0432]: unresolved import `syn::export`
   --> <user>/.cargo/registry/src/github.com-1ecc6299db9ec823/pgx-utils-0.1.11/src/pg_config.rs:250:10
    |
250 | use syn::export::Formatter;
    |          ^^^^^^ could not find `export` in `syn`
```

**Comment by chou666** [comment]
same issue here

**Comment by hcyrs** [comment]
same issue

**Comment by mathroc** [comment]
Cargo.lock should probably be added to the repo to avoid issue like this and #66

**Comment by eeeebbbbrrrr** [comment]
Thanks for the reports y'all.  This is now affecting me too!  It's like a little virus.

I'll do two things -- upgrade to the latest syn and fix the actual problem, and then I'll lock it all down `Cargo.lock`.  I'll try to get a new release published today, but tomorrow (Jan 8, 2021) at the latest.

Hang in there!

**Comment by eeeebbbbrrrr** [comment]
should be fixed in 0.1.13.

**Comment by chou666** [comment]
Thanks it works !
